src/commands/lore/lore.ts

```diff
@@ -60,16 +60,15 @@ export default class Lore implements ICommand {
         return {
             type: "embed",
             result: embed,
-        }
+        };
     }
 
     async run(ctx: IContext) {
         const name = ctx.getOption("name")?.toLowerCase() || "";
         const output = await this.getOutput(name);
-        if (output.type === "text"){
-            ctx.replyText(output.result as string)
-            return
-        }
-        ctx.replyEmbed(output.result as Embed);
+        if (output.type === "text")
+            ctx.replyText(output.result as string);
+        else
+            ctx.replyEmbed(output.result as Embed);
```

Comment:
Please add curly braces in the if statement. In that case, it makes the code less maintainable

(sorry for not documenting the code style)